tap_jira/streams.py

```diff
@@ -177,8 +169,9 @@ def sync(self, ctx):
             comments = []
             for issue in page:
                 comments += issue["fields"].pop("comment")["comments"]
-            ISSUE_COMMENTS.format_comments(comments)
-            ISSUE_COMMENTS.write_page(comments)
+            if ISSUE_COMMENTS.tap_stream_id in ctx.selected_stream_ids:
+                ISSUE_COMMENTS.format_comments(comments)
+                ISSUE_COMMENTS.write_page(comments)
```

Comment:
This is just cleaning up the "is selected" check. Previously, when `ISSUE_COMMENTS` called the `write_page` function, the issue comments were just not written to stdout. I removed the check from `write_page` because I think things just make a little more sense this way.